app/1.0/blog.yaml

```diff
@@ -1,7 +1,15 @@
 ---
+- title: "Polymer News Progressive Web App"
+  path: /1.0/blog/2017-02-06-news-pwa
+  published: 2017-02-06
+  author: polymer-team
+  description: "The new Polymer News demo progressive web app helps you start building a cutting-edge PWA using Polymer."
+  tag:
+    - blog
+    - announcements
 - title: "webcomponents.org"
-  path: /1.0/blog/2017-01-09-webcomponents-org
-  published: 2017-01-09
+  path: /1.0/blog/2017-01-10-webcomponents-org
+  published: 2017-01-10
```

Comment:
Good point - apologies. This was an old commit hanging around that I never ended up pushing. Will remove.